srcpkgs/Logseq/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'Logseq'
+pkgname=Logseq
+version=0.9.8
+revision=1
+archs="x86_64"
+short_desc="Open-source platform for knowledge sharing and management"
+maintainer="Viachaslau Ravinski <shnapsx@gmail.com>"
+license="AGPL-3.0-only"
+homepage="https://github.com/logseq/logseq"
+distfiles="https://github.com/logseq/logseq/releases/download/${version}/logseq-linux-x64-${version}.zip"
+checksum=6ccea0db1c7083c72046fcba80ac2e80cfa16196e10280fcb0438d652ee119d2
+nostrip_files="Logseq"
+
+do_install() {
+	vinstall "${FILESDIR}/Logseq.desktop" 644 usr/share/applications
+	vinstall "${XBPS_BUILDDIR}/${pkgname}-${version}/resources/app/icons/logseq.png" 644 usr/share/pixmaps logseq.png
+	vcopy "${XBPS_BUILDDIR}/${pkgname}-${version}" usr/lib/Logseq
+	vlicense "${XBPS_BUILDDIR}/${pkgname}-${version}/LICENSE"
+	vlicense "${XBPS_BUILDDIR}/${pkgname}-${version}/LICENSES.chromium.html"
```

Comment:
the first argument of most of the v* functions are relative to `$wrksrc`, which is `$XBPS_BUILDDIR/$pkgname-$version`

Reply:
Not sure what you suggest here. Could you tell me more?

Reply:
`vlicense "${XBPS_BUILDDIR}/${pkgname}-${version}/LICENSE"` should be `vlicense LICENSE`

Reply:
Updated this template to compile from sources but seems like it requires openssl 3. 
```
SONAME: libssl.so.3 <-> UNKNOWN PKG PLEASE FIX!
SONAME: libcrypto.so.3 <-> UNKNOWN PKG PLEASE FIX!
```
Is there any way I can fix it until https://github.com/void-linux/void-packages/pull/37681 is merged?